test/integration/util/util.go

```diff
@@ -120,7 +122,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
-	p, _ := exec.LookPath("oc")
```

Comment:
This will not work because as per doc LookPath only able to search binary if that is part of $PATH https://golang.org/pkg/os/exec/#example_LookPath